src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -209,7 +209,7 @@ public int getOstrichPort() {
     }
 
     public String getCloudService() {
-        return getString("cloud.service");
+        return getString("cloud.service", "");
```

Comment:
It felt useless to me; but you are right this is not necessary to put this default value; I'll change it